src/openrct2/park/ParkFile.cpp

```diff
@@ -867,6 +867,11 @@ namespace OpenRCT2
                 // Invention list
                 cs.ReadWriteVector(gResearchItemsUninvented, [&cs](ResearchItem& item) { ReadWriteResearchItem(cs, item); });
                 cs.ReadWriteVector(gResearchItemsInvented, [&cs](ResearchItem& item) { ReadWriteResearchItem(cs, item); });
+
+                if (cs.GetMode() == OrcaStream::Mode::READING)
+                {
+                    research_determine_first_of_type();
+                }
```

Comment:
Might be better to move this to the Import method, before the call to fix_game_save_vars.